### ISSUE TITLE
Index panel moves to wrong position when clicking link

### ISSUE THREAD

**Issue by tonylukasavage** [docstrap/docstrap#37]
I have a basic class doc that looks like this:

![original](http://cl.ly/image/0w182A2l3Q3K/Screen%20Shot%202014-04-09%20at%202.06.13%20PM.png)

In the description of the `doc` and `file` properties I make reference to the `parse` function at the end of the docs. The link is defined as `{@link Tiapp#parse|parse()}`. If I click on the link, ink-docstrap navigates to the correct section as best it can, but the selected index in the table of contents on the right is wrong, as seen here:

![bad](http://cl.ly/image/1x402E3a1J0z/Screen%20Shot%202014-04-09%20at%202.06.22%20PM.png)

I assume this is because there's not enough content on the page for ink-docstrap to scroll down nay further, given that `parse()` is the last API on the page, and it's using purely the scroll position to determine where the selected index is on the TOC. To confirm this suspicion I added a dummy API below `parse()`. With an adequately large dummy API below, the selected index works as expected, as seen here: 

![correct](http://cl.ly/image/2t2h2n3a1r2Y/Screen%20Shot%202014-04-09%20at%202.08.21%20PM.png)

So in short, regardless of the actual scroll position, I need the selected index on the table of contents on the right to reflect the link that is clicked on. If I click on `Tiapp#parse`, the selected index should be for `parse()`, not the section above it.
